static/sass/_patterns_blog-card.scss

```diff
@@ -0,0 +1,130 @@
+@mixin  blog-p-card {
+
+  .blog-p-card__category {
+    background: url('#{$assets-path}ed42aefa-icon-resource-hub-icon-document.png') left center no-repeat;
+    color: $color-mid-dark;
+    font-size: $sp-medium;
+    line-height: 1.5;
+    padding: 0 0 0 $sp-large;
+    text-transform: uppercase;
+
+    > a:link,
+    > a:visited {
+      color: $color-mid-dark;
+      text-decoration: none;
+    }
+
+    > a:hover,
+    > a:active {
+      color: $color-brand;
+      text-decoration: underline;
+    }
+  }
+
+  %post-card-header {
+    border-radius: 2px;
+    padding: $sp-medium $sp-medium $sp-small;
+  }
+
+  .blog-p-card--post {
+    @extend .p-card--highlighted;
+    display: flex !important;
+    flex-direction: column;
+    padding: 0;
+
+    .p-strip--featured & {
+      background-color: rgba(255, 255, 255, .9);
+      position: relative;
+      z-index: 1;
+    }
+
+    > .blog-p-card__content {
+      border-top: 1px dotted $color-mid-light;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      margin: 0 $sp-small;
+      padding: $sp-medium $sp-small $sp-small;
+
+      > a,
+      h3,
+      p {
+        display: block;
+        margin: $sp-small 0;
+      }
+    }
+
+    .blog-p-card__footer {
+      border-top: 1px dotted $color-mid-light;
+      font-size: .875rem;
+      margin: auto $sp-small 0;
+      max-width: inherit;
+      padding: $sp-medium $sp-small;
+    }
+  }
+
+  .blog-p-card--muted {
+    @extend .p-card;
+    background-color: $color-light;
+    padding: 0;
+    box-shadow: 0 1px 2px 0 transparentize($color-dark, .8);
+
+    > .blog-p-card__header  {
+      margin-bottom: 0;
+      border-top: 3px solid $color-light;
+      border-bottom: 0;
+    }
+
+    > .blog-p-card__content {
+      margin: 0 $sp-small $sp-small;
+      padding: $sp-medium $sp-small $sp-small;
+      border-top: 1px dotted $color-mid-light;
+    }
+  }
+
+  .blog-p-card__header {
+    @extend %post-card-header;
+
+    &--cloud-and-server {
+      @extend %post-card-header;
+      border-top: 3px solid #a87ca0;
```

Comment:
Why are we using custom hex values here?

Reply:
I took the code from https://github.com/canonical-websites/blog.ubuntu.com/blob/master/static/sass/_pattern_card.scss . 
Did not update them. 
I could create an Issue for this to fix it in another PR.